analyses/gene_match/README.md

```diff
@@ -1,4 +1,7 @@
-## To run, 
+## Gene symbol matching across GENCODE versions
+This analysis is DEPRECATED and was last run with OpenPedCan data release `v12`.
+
+To run, 
```

Comment:
It would be better to add "## " in front of To run to make it more obvious to see.

Reply:
done